content/ADR-202-linked-wearables.md

```diff
@@ -0,0 +1,212 @@
+        ]
+      }
+    }
+  }
+```
+
+- **Owns items from different collections**: Any user who owns an item with an id between `100` and `200`, or the specific ids `203`, `207`, and `233` from the collection `0x1234567890abcdef1234567890abcdef12345678` on `matic`, or who owns any item from the collection `0xabcdefabcdefabcdefabcdefabcdefabcdef12345678` on `mainnet`, will be granted this wearable.
+
+```
+  {
+    "mappings": {
+      "matic": {
+        "0x1234567890abcdef1234567890abcdef12345678": [
+          {
+            "type": "range",
+            "from": "100",
+            "to": "200"
+          },
+          {
+            "type": "multiple",
+            "ids": ["203", "207", "233"]
+          }
+        ]
+      },
+      "mainnet": {
+         "0xabcdefabcdefabcdefabcdefabcdefabcdef12345678": [
+          {
+            "type": "any"
+          }
+        ]
+      }
+    }
+  }
+```
+
+### Profile and Outfits Deployments
+
+The LinkedWearable entity represents sets of NFT collection items, but users own individual NFTs within those collections. When users interact with the wearable to equip it in their profile or outfit, they do so with a single item from the collection. To facilitate this, an extended URN is used to identify the specific item a user is interacting with. This extended URN begins with the identification of the wearable, as stated in the previous section, and continues with the data that identifies the NFT collection item granting the wearable. This information allows the Catalyst node to perform on-chain validations, verifying that the user's wallet owns the NFT associated with the wearable.
+
+*The use of extended URNs for the individuals items management is documented on [ADR-244](https://adr.decentraland.org/adr/ADR-244).*
+
+- **LinkedWearables URN in a Profile or Outfit Entity**: `urn:decentraland:matic:collections-thirdparty:[third-party-name]:[collection-name]:[item-name]:[chain]:[contract]:[tokenId]`
+   - **chain**: Chain where the NFT collection contract is hosted 
+   - **contract**: Address of an ERC-721 or ERC-1155 collection contract 
+   - **tokenId**: collection item id 
+
+Example URN: `urn:decentraland:matic:collections-thirdparty:adidas:summer2024:red-hoodie:matic:0xba0c9cf4da821dba98407cc4f9c11f6c7a5f9bbc:1`
+
+With the information from the URN, the collection contract at address `0xba0c9cf4da821dba98407cc4f9c11f6c7a5f9bbc` on `matic`, can be queried to verify if the user equipping the wearable owns the item with id `1`.
+
+### Third Party Registry 
+
+When a third party is registered, a record is added to the third party's smart contract. This record must now include the list of NFT collection contracts and their corresponding chains for which the third party will create wearables. This information is essential for the Catalyst nodes to create a backpack endpoint, retrieve all NFTs owned by a wallet across those collections, and identify any corresponding linked wearables for those NFT items.
```

Comment:
Can we add an example?

Reply:
It should be

`tp:version:lw-name:lw-description:lw-collections`

E.g: `tp:1:third-party-test:this-is-a-description:sepolia-0x74c78f5A4ab22F01d5fd08455cf0Ff5C3367535C;amoy-0x31c78f3A4ac21F41e5ad28345df1Fa5A3367213A`